manifests/cf-manifest/deployments/060-aws-broker.yml

```diff
@@ -0,0 +1,61 @@
+releases:
+  - name: aws-broker
+    version: v3
```

Comment:
Note that this version naming is inconsistent with the other PR: https://github.com/alphagov/paas-cf/pull/221/files#diff-b3a119020efd88e25d5115307a683e4bR32

IMO we should change one of them, regardless other releases are using a different version naming (logsearch/graphite use commit ID)